packages/samples/headless-ssr/common/components/common/search-engine-config.ts

```diff
@@ -14,7 +14,10 @@ import {
 export const config = {
   configuration: {
     ...getSampleSearchEngineConfiguration(),
-    analytics: {enabled: false},
+    analytics: {
+      analyticsMode: 'next',
+      trackingId: 'sports-ui-samples-legacy',
```

Comment:
This trackingId does not exist in searchuisamples, the only trackingId that's defined and associated with a catalog is `sports-ui-samples`